provider/lib/utils.js

```diff
@@ -98,6 +98,9 @@ module.exports = function(
         var method = 'postTrigger';
 
         return new Promise(function(resolve, reject) {
+            var triggerIdentifier = utils.getTriggerIdentifier(dataTrigger.apikey, dataTrigger.namespace, dataTrigger.name);
```

Comment:
there is no reason to print the debug statement here.  It can be done in the caller method (fireTrigger method right before the call to postTrigger on line 85) and have the same effect.  In that method we already have the trigger ID as well.  I would also change the message to something like the following:
logger.info(method, 'Cron fired for trigger', triggerId)